src/modules/conditions.js

```diff
@@ -124,6 +126,10 @@ export const unprefixOperator = (operator) => {
     }
 }
 
+// TODO - in practice this function isn't used for the 'IN' operator
+// but if it were the result would be wrong. The function
+// should probably control for the allowed operators and throw if the
+// operator isn't one of the allowed ones.
```

Comment:
Good point. You could add a check in the beginning of the function like 
`if(!['EQ','!EQ'.....].includes(operator) {throw 'some message...'}` and add the error to the `expected` prop for the `IN` case in the tests above.

Reply:
I'll address this in a later PR since this increases the code and test scope somewhat (handling potential thrown errors).